### PR TITLE
Updating documentation for the relevancy feed

### PR DESCRIPTION
**Pull request by jeremyf**
## What type of PR is this? (check all applicable)

- [x] Documentation Update

## Description

This looks to remove the chatter of the document and instead point folks
to locations within the code.

## Related Tickets & Documents

Closes forem/forem#17245

## QA Instructions, Screenshots, Recordings

None.

### UI accessibility concerns?

None.

## Added/updated tests?

- [x] No, and this is why: This is documentation only.

## [Forem core team only] How will this change be communicated?

- [x] I will share this change internally with the appropriate teams

### CODE REVIEW

File: app/models/articles/feeds/README.md [hunk 1]
```diff
@@ -1,128 +1,39 @@
...
 
-There are three layers of configuration, presented working from the inside out.
+## So You Want to Understand the Variant Query
 
-1.  Lever
-2.  Feed Strategy
-3.  Feed Experiment
+In
+[Diving into Dev's Relevancy Feed Builder](https://dev.to/devteam/diving-into-devs-relevancy-feed-builder-30m6)
+we explained the conceptual implementation; providing conceptual entity
```
Comment (juliannatetreault) [inline app/models/articles/feeds/README.md:33]: Quick, non-blocking suggestion: should this actually read as written below?
```suggestion
we explained the conceptual implementation; providing a conceptual entity
```

**Review by juliannatetreault — APPROVED** [review]

File: app/models/articles/feeds/README.md [hunk 2]
```diff
@@ -1,128 +1,39 @@
...
-  [Articles::Feeds::WeightedQueryStrategy](https://github.com/forem/forem/blob/de2edee07d824a34e5c5445d455b1f8086bd127d/app/services/articles/feeds/weighted_query_strategy.rb)
-  is the precursor to the more robust feed configuration. It provides hints as
-  to how we'll structure the feed configuration.
-- [These Are the [Feed] Levers I Know I Know - Forem Team 🌱](https://dev.to/devteam/these-are-the-feed-levers-i-know-i-know-3jj7)
-- [Creating a Catalog of Feed Levers and Configurations - Forem Team 🌱](https://forem.team/jeremy/creating-a-catalog-of-feed-levers-and-configurations-nnk)
-- On <2022-04-11 Mon>,
-  [Jeremy, Jennie, Ella, and Michael T met over Zoom to talk through the current feed](https://forem.team/jeremy/lets-talk-about-the-feed-baby-54b1).
+  [Articles::Feeds::WeightedQueryStrategy](https://github.com/forem/forem/blob/de2edee07d824a34e5c5445d455b1f8086bd127d/app/services/articles/feeds/weighted_query_variant.rb)
+  is the precursor to the more robust feed configuration.
+- [These Are the [Feed] Levers I Know I Know](https://dev.to/devteam/these-are-the-feed-levers-i-know-i-know-3jj7)
```
Comment (msarit) [inline app/models/articles/feeds/README.md:11]:
```suggestion
- [This post](https://dev.to/devteam/these-are-the-feed-levers-i-know-i-know-3jj7)
```
(nitpick) I appreciate the wittiness of the post title 😄  but I think it's too long for documentation.

File: app/models/articles/feeds/README.md [hunk 3]
```diff
@@ -1,128 +1,39 @@
...
+  walked through the meaning of each of the initial relevancy levers.
 
-## Glossary
+In addition there are two code walk throughs explaining the two primary pull
```
Comment (msarit) [inline app/models/articles/feeds/README.md:14]:
```suggestion
In addition there are two code walk throughs explaining the pull
```

File: app/models/articles/feeds/README.md [hunk 4]
```diff
@@ -1,128 +1,39 @@
...
-At a future point, we might allow for the site creators to pick one or more feed
-strategies for the _relevancy feed_ and even allow a member to specify their
-feed. It is possible that we would adjust the UI so that the feed formats
-(Relevant, Recent, and Top) also include things like "Active Discussions" or
-other emergent _feed strategies_.
+## So You Want to Change the Relevancy Feed
 
-## Configuration
+The [forem/forem#17406 pull request](https://github.com/forem/forem/pull/17406)
+contains an example of constructing a
```
Comment (msarit) [inline app/models/articles/feeds/README.md:23]:
```suggestion
contains an example of configuring a
```

File: app/models/articles/feeds/README.md [hunk 5]
```diff
@@ -1,128 +1,39 @@
...
+[Articles::Feeds::VariantQuery](https://github.com/forem/forem/blob/main/app/services/articles/feeds/variant_query.rb)
```
Comment (msarit) [inline app/models/articles/feeds/README.md:24]:
```suggestion
[VariantQuery](https://github.com/forem/forem/blob/main/app/services/articles/feeds/variant_query.rb)
```

File: app/models/articles/feeds/README.md [hunk 6]
```diff
@@ -1,128 +1,39 @@
...
+configuration. You can also read the
```
Comment (msarit) [inline app/models/articles/feeds/README.md:25]:
```suggestion
You can also read the
```

File: app/models/articles/feeds/README.md [hunk 7]
```diff
@@ -1,128 +1,39 @@
...
+[config/feed-variants/README.md](https://github.com/forem/forem/blob/main/config/feed-variants/README.md)
+for further context.
 
-There are three layers of configuration, presented working from the inside out.
+## So You Want to Understand the Variant Query
```
Comment (msarit) [inline app/models/articles/feeds/README.md:29]:
```suggestion
## Understanding the Variant Query
```

File: app/models/articles/feeds/README.md [hunk 8]
```diff
@@ -1,128 +1,39 @@
...
-constraint. This can generate a large and potentially expensive query, so we
-want to discard older articles from the relevancy.
-
-What is the recommended size of that _published_at_ constraint? It is a function
-of the average number of posts per week. A more active Forem, such as
-[DEV](https://dev.to) would want to limit this _published_at_ constraint to
-around 2 weeks. A less active one could limit to 2 months.
+The key thing to internalize is as follows: We want to create a SQL statement
+that calculates a `relevancy_score` for a given user and each article in the
+Forem instance. The higher the `relevancy_score` an article has, the closer to
```
Comment (msarit) [inline app/models/articles/feeds/README.md:38]:
```suggestion
Forem instance. The higher the article's `relevancy_score`, the closer that article will be to
```

File: app/models/articles/feeds/README.md [hunk 9]
```diff
@@ -1,128 +1,39 @@
...
+the top that article will be.
```
Comment (msarit) [inline app/models/articles/feeds/README.md:39]:
```suggestion
the top of the feed.
```

**Review by msarit — APPROVED** [review]
Made some non-opinionated edits, mostly in sentence structure and flow. I'm alright with the original text remaining as-is. 👍🏾

File: app/models/articles/feeds/README.md [hunk 10]
```diff
@@ -1,128 +1,39 @@
...
 - The
-  [Articles::Feeds::WeightedQueryStrategy](https://github.com/forem/forem/blob/de2edee07d824a34e5c5445d455b1f8086bd127d/app/services/articles/feeds/weighted_query_strategy.rb)
-  is the precursor to the more robust feed configuration. It provides hints as
-  to how we'll structure the feed configuration.
-- [These Are the [Feed] Levers I Know I Know - Forem Team 🌱](https://dev.to/devteam/these-are-the-feed-levers-i-know-i-know-3jj7)
-- [Creating a Catalog of Feed Levers and Configurations - Forem Team 🌱](https://forem.team/jeremy/creating-a-catalog-of-feed-levers-and-configurations-nnk)
-- On <2022-04-11 Mon>,
-  [Jeremy, Jennie, Ella, and Michael T met over Zoom to talk through the current feed](https://forem.team/jeremy/lets-talk-about-the-feed-baby-54b1).
+  [Articles::Feeds::WeightedQueryStrategy](https://github.com/forem/forem/blob/de2edee07d824a34e5c5445d455b1f8086bd127d/app/services/articles/feeds/weighted_query_variant.rb)
+  is the precursor to the more robust feed configuration.
```
[the remark this answers is not in the document]
Comment (jeremyf) [reply app/models/articles/feeds/README.md:10]: From an archaeological stand point, the WeighedQueryStrategy came first and we evolved the code to use the VariantQuery then removed the WeightedQueryStrategy.

**Pull request merged**